server.go

```diff
@@ -577,11 +588,17 @@ func (s *Server) HandleTracePacket(packet []byte) {
 			s.Statsd.Count("packet.error_total", 1, []string{"packet_type:ssf_metric", "reason:parse"}, 1.0)
 			return
 		}
-		s.Workers[metric.Digest%uint32(len(s.Workers))].PacketChan <- *metric
+		if ValidMetric(*metric) {
+			s.Workers[metric.Digest%uint32(len(s.Workers))].PacketChan <- *metric
+		} else {
+			s.Statsd.Count("packet.error_total", 1, []string{"packet_type:ssf_metric", "reason:not_valid"}, 1.0)
```

Comment:
I double-checked this and found that we use `metric` as the tag value for this in other places.